tbans/tbans_service.py

```diff
@@ -0,0 +1,58 @@
+import logging
+
+from protorpc import remote
+from protorpc.wsgi import service
+
+from models.service.messages import TBANSResponse, PingRequest, VerificationRequest, VerificationResponse
+
+
+package = 'tbans'
+
+
+class TBANSService(remote.Service):
+    """ The Blue Alliance Notification Service.... Service """
+
+    @remote.method(PingRequest, TBANSResponse)
+    def ping(self, request):
+        """ Immediately dispatch a Ping to either FCM or a webhook """
+        if request.fcm and request.webhook:
+            return TBANSResponse(code=400, message='Cannot ping both FCM and webhook')
```

Comment:
I'm not familiar enough with RPC responses - this feels like we should be able to set a status code for the request as opposed to just setting a code in an object, since this will 200 and we'll lose some clarity around what's going on in logs/debugging